src/data/services/DiscoveryDataApiService.js

```diff
@@ -17,6 +17,19 @@ class DiscoveryDataApiService {
     });
   }
 
+  static fetchCourseRuns(params) {
```

Comment:
This feels like a weird function name when it's returning a single course_uuid. You aren't fetching any course runs. Maybe `fetchCourseThroughCourseRun` or something like that, but `fetchCourseRuns` feels weird.

Reply:
Also shouldn't this more closely follow the above method? We are only sending in a single course run, right? Why not make the path `${DiscoveryDataApiService.discoveryBaseUrl}/course_runs/${key}`?

Reply:
Your totally right, didn't realize the course run detail accepted a key